File: test/Video/test05.py
#!/usr/bin/python
from ming import *
import sys

srcdir=sys.argv[1]

Ming_useSWFVersion(7);
m =  SWFMovie();

stream = SWFVideoStream(srcdir + "/../Media/video01.flv");

mc = SWFMovieClip();
mc.add(stream);

numFrames = stream.getNumFrames()
for i in range(numFrames):
	mc.nextFrame();

m.add(mc);
m.nextFrame();


m.save("test05.swf");

File: test/Video/test05.php
#!/usr/bin/php -c.
<?php

$srcdir = $argv[1];

$m = new SWFMovie(7);

$stream = new SWFVideoStream($srcdir . "/../Media/video01.flv");
$numFrames = $stream->getNumFrames();

$mc = new SWFMovieClip();
$mc->add($stream);
for($i = 0; $i < $numFrames; $i++)
{
	$mc->nextFrame();
}

$m->add($mc);
$m->save("test05.swf");

?>
